cmd/talosctl/cmd/talos/edit.go

```diff
@@ -0,0 +1,168 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package talos
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"runtime"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/cmd/util/editor"
+	"k8s.io/kubectl/pkg/cmd/util/editor/crlf"
+
+	"github.com/talos-systems/talos/cmd/talosctl/pkg/talos/helpers"
+	"github.com/talos-systems/talos/pkg/machinery/api/machine"
+	"github.com/talos-systems/talos/pkg/machinery/client"
+)
+
+var editCmdFlags struct {
+	namespace string
+}
+
+// editCmd represents the edit command.
+var editCmd = &cobra.Command{
+	Use:     "edit <RESOURCE/NAME>",
+	Aliases: []string{"apply"},
```

Comment:
I feel `apply` shouldn't be alias for `edit`... 

What I thought is to provide alias for `apply-config` as `apply config` if we introduce `edit config`

Reply:
ah, gotcha